ScreenBoom/ScreenBoom/ViewController/PlayEventViewController.swift

```diff
@@ -121,18 +127,18 @@ class PlayEventViewController: BaseViewController, PlayEventViewModelSourceObser
     var playView: PlayEventView?
     
     switch (self.event.eventType) {
-      case .Text:
-        playView = TextPlayEventView()
-        break
-      case .Photo:
-        playView = PhotoPlayEventView()
-        break
-      case .Animation:
-        playView = AnimationPlayEventView()
-        break
-      case .Unknown:
-        playView = nil
-        break
+    case .Text:
+      playView = TextPlayEventView()
```

Comment:
why are we making them empty ? cant we just make them nil ?

Reply:
looks like this, 

`self.subviews.forEach({ $0.removeFromSuperview; $0 = nil })`